template/package.json

```diff
@@ -7,17 +7,17 @@
   "scripts": {
     "clean": "rimraf lib",
     "precompile": "npm run clean",
-    "compile": "webpack --config ./config/webpack.prod.config.js",
+    "compile": "webpack --config ./config/webpack.prod.config.js --progress",
     "start": "webpack-dev-server --config ./config/webpack.dev.config.js",
     "start:watch": "npm run clean && webpack --config ./config/webpack.watch.config.js",
-    "start:prod": "npm run compile && electron --noDevServer ./electron",
     "test": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\"",
     "test:watch": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\" --watch",
     "test:karma": "cross-env NODE_ENV=test karma start",
     "cover": "nyc npm run test",
     "ci": "npm run check && npm test",
     "fmt": "mongodb-js-fmt ./*.js ./test/*.js",
-    "check": "mongodb-js-precommit './src/**/*{.js,.jsx}' './test/**/*.js' index.js",
+    "precheck": "npm run compile",
```

Comment:
is compilation necessary as a precheck step if the check itself only runs in ./src and ./test?

Reply:
as discussed. needs to be done because check also goes through the package.json main entry point.